session_security/static/session_security/script.js

```diff
@@ -66,7 +66,12 @@ yourlabs.SessionSecurity.prototype = {
 
     // Called by click, scroll, mousemove, keyup.
     activity: function() {
-        this.lastActivity = new Date();
+        var now = new Date();
+        if (now - this.lastActivity < 1000)
```

Comment:
Do you mean the lag of hiding the "Your session is about to expire" warning and dispatching the ping? I might be missing something obvious, but I don't think this can be noticeable to user. 

Suppose the warning is visible on screen and user moves the mouse, and we're concerned the might be ~1s delay until the warning disappears. But, the warning being on the screen means the user has been idle for a good while. So the (now - this.lastActivity < 1000) check would fall through.